cloudinary/utils.py

```diff
@@ -1037,6 +1037,28 @@ def build_upload_params(**options):
     return params
 
 
+def build_multi_and_sprite_params(**options):
+    """
+    Build params for multi, download_multi, generate_sprite, and download_generated_sprite methods
+    """
+    tag = options.get("tag")
+    urls = options.get("urls")
+    if tag and urls:
```

Comment:
you can replace these two validations with XOR, saying 
"Either 'tag' or 'urls' parameter has to be set but not both"